samples/nrf9160/modem_shell/src/curl/tool/CMakeLists.txt.orig

```diff
@@ -0,0 +1,114 @@
+#***************************************************************************
```

Comment:
I will remove this file, right?
@jhirsi

Reply:
Actually, might be ok to keep also and change the file type extension to differ to one that is usually generated e.g. within git merges. That is the CMakelist of the Curl and might be good to keep in as a reference.

Reply:
Ok, let's see what other reviewers say about this.

Reply:
If you think it's useful for you I have no issues keeping it.